boards/arm/stm32/stm32f4discovery/configs/wifi/defconfig

```diff
@@ -20,7 +20,6 @@ CONFIG_ARCH_CHIP_STM32=y
 CONFIG_ARCH_CHIP_STM32F407VG=y
 CONFIG_ARCH_INTERRUPTSTACK=2048
 CONFIG_ARCH_STACKDUMP=y
-CONFIG_ARMV7M_LAZYFPU=y
```

Comment:
Probably need to remove `CONFIG_TESTING_OSTEST_FPUSIZE=y` here and in some other configs where `CONFIG_ARMV7M_LAZYFPU=y` is removed.

Reply:
@anchao will provide a new patch improving fpu test design:

1. Remove CONFIG_TESTING_OSTEST_FPUSIZE config
2. Merge related code from board/chip to arch common

Let's wait his PR.

Reply:
@anchao 's patch is merged, let's continue this PR.